src/PantosCoinWrapper.sol

```diff
@@ -37,6 +37,7 @@ contract PantosCoinWrapper is PantosWrapper {
      */
     function unwrap(uint256 amount) public override whenNotPaused onlyNative {
         _burn(msg.sender, amount);
-        payable(msg.sender).transfer(amount);
+        (bool success, ) = payable(msg.sender).call{value: amount}("");
+        require(success, "PantosCoinWrapper: transfer failed");
```

Comment:
Can we also have a unit test for this require? Not so important since it's a trivial operation but it would be nice to have for coverage completeness